index.html

```diff
@@ -0,0 +1,1685 @@
+                    presidential nomination.</p>
+                </section>
+              </article>
+            </div>
+
+            <div class="col-xl-3 col-md-6">
+              <article>
+                <section class="posit-rel">
+                  <div class="category">
+                    <a class="red-tx" href="#">POLITICS</a></div>
+                  <div class="main-row-img-1">
+                    <img src="assets/images/main1row2.jpg" alt="description">
+                  </div>
+                </section>
+                <section class="main-row-txt-1">
+                  <h2><a href="#"> Bloomberg Could Become 2020 Heavyweight
+                      Immediately, Political Scientist Says</a></h2>
+                  <p>The former New York City mayor is reportedly on the verge
+                    of entering the Democratic party's race for the
+                    presidential nomination.</p>
+                </section>
+              </article>
+            </div>
+
+            <div class="col-xl-3 col-md-6">
+              <article>
+                <section class="posit-rel">
+                  <div class="category">
+                    <a class="red-tx" href="#">POLITICS</a></div>
+                  <div class="main-row-img-1">
+                    <img src="assets/images/main1row3.jpg" alt="description">
+                  </div>
+                </section>
+                <section class="main-row-txt-1">
+                  <h2><a href="#"> Bloomberg Could Become 2020 Heavyweight
+                      Immediately, Political Scientist Says</a></h2>
+                  <p>The former New York City mayor is reportedly on the verge
+                    of entering the Democratic party's race for the
+                    presidential nomination.</p>
+                </section>
+              </article>
+            </div>
+
+            <div class="col-xl-3 col-md-6">
+              <article>
+                <section class="posit-rel">
+                  <div class="category">
+                    <a class="red-tx" href="#">POLITICS</a></div>
+                  <div class="main-row-img-1">
+                    <img src="assets/images/main2row1.jpg" alt="description">
+                  </div>
+                </section>
+                <section class="main-row-txt-1">
+                  <h2><a href="#"> Bloomberg Could Become 2020 Heavyweight
+                      Immediately, Political Scientist Says</a></h2>
+                  <p>The former New York City mayor is reportedly on the verge
+                    of entering the Democratic party's race for the
+                    presidential nomination.</p>
+                </section>
+              </article>
+            </div>
+
+          </div>
+        </section>
+
+
+        <section class="col-12">
+          <div class="title-generic">
+            <img src="assets/images/NlogoS.png" alt="description">
```

Comment:
The image is not showing please correct the image name `NLogoS.png` instead of `NlogoS .png`

Reply:
Ok, thanks